tests/unit/amo/containers/TestHome.js

```diff
@@ -6,21 +6,21 @@ import {
 } from 'react-addons-test-utils';
 import { Provider } from 'react-redux';
 
-import createStore from 'amo/store';
+import { HomeBase } from 'amo/containers/Home';
+import { dispatchSignInActions } from 'tests/unit/amo/helpers';
 import { getFakeI18nInst } from 'tests/unit/helpers';
-import Home from 'amo/containers/Home';
 
 
 describe('Home', () => {
   it('renders a heading', () => {
-    const initialState = { api: { clientApp: 'android', lang: 'en-GB' } };
-    const { store } = createStore(initialState);
+    const { store } = dispatchSignInActions();
+    const fakeDispatch = sinon.stub();
 
     const root = findRenderedComponentWithType(renderIntoDocument(
       <Provider store={store}>
-        <Home i18n={getFakeI18nInst()} />
+        <HomeBase dispatch={fakeDispatch} i18n={getFakeI18nInst()} />
```

Comment:
This can just be `dispatch={store.dispatch}`. I think it's risky to mix real dispatches (via `dispatchSignInActions()`) with a stub dispatcher. 

I think this was ghost code anyway. The fake dispatcher doesn't appear to be used anymore.